### PR TITLE
Doc test show newest products slide

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Api documents for show newest products slide
- Test Api
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Feature/Api/ShowNewestProductsSlideTest.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+     * @param array  $structure structure of json 
+     *
+     * Test api structure
+     *
+     * @return void
+     */
+    public function test_json_structure($url, $structure)
+    {
+        $response = $this->json('GET', $url);
+        //dd($response);
```
Comment (at-hienpham) [inline tests/Feature/Api/ShowNewestProductsSlideTest.php]: xóa dd đi vỹ

File: tests/Feature/Api/ShowNewestProductsSlideTest.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        factory(Product::class)->create();
+        factory(Image::class)->create();
+    }
+
+    /**
+     * Test status code
+     *
+     * @return void
+     */
+    public function test_status_code_for_show_newest_products_sldie()
```
Comment (at-hienpham) [inline tests/Feature/Api/ShowNewestProductsSlideTest.php]: sửa lại thành slide :v

File: tests/Feature/Api/ShowNewestProductsSlideTest.php [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
```
Comment (at-quangtran) [inline tests/Feature/Api/ShowNewestProductsSlideTest.php]: test_status_code

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**